web-console/1-homepage/project-panel.md

```diff
@@ -1,63 +1,98 @@
-*******************************
-Things you may want to include in this file
-Getting started (day 1)
-  - Create New Project
-Project view (day 2)
-  - Card View
-After Day 2
-  - Project Summary
-  - List View
+# Project Panel
+- The project panel should cover 1/3 of the homepage when viewing on a standard desktop screen.
+- The panel is on the far right-hand side for launch.
+- **Note:** In the future the panel size and location may be configurable by the user.
 
+## Day 1 Experience
+![template](img/OpenShiftMall_Day1-ProjectsBar-1B.png)
 
-*******************************
+- Brand new users will not have any projects upon initial login.
+- On day 1, the projects panel includes the following sections in this order:
+  1. **Create Project Button** (optional): The create project button is displayed by default, but admin users may choose to hide this button if desired.
+  2. **Guided Tour:** Action button that launches a quick tutorial to identify key sections of the interface.
+    * Details TBD
+  3. **Resources:** Additional links that may help user learn more about the product will be listed here.
+    * Details TBD
+  4. **Custom Text/Links** (optional): This section is optional and configurable by the admin user. It allows admin users to provide additional content for their day-to-day users.
```

Comment:
Wonder if it makes sense for us to focus on Day 1 VS Day 2 in the documentation?  We show additional information if projects exists, and if they don't exist, we may add extra documentation links.  I wonder from a design doc perspective if it makes more sense to just say that?